tutorials/colors/colormaps.py

```diff
@@ -116,10 +151,10 @@
 # banding of the data in those values in the colormap (see [mycarta-banding]_ for
 # an excellent example of this).
 
-cmaps['Sequential (2)'] = [
+plot_color_gradients('Sequential (2)', [
```

Comment:
Indent following lines to inside `(`.

Reply:
I was trying to avoid changing too many lines there, but sure.